exec/java-exec/src/main/java/org/apache/drill/exec/store/ActionOnFile.java

```diff
@@ -0,0 +1,68 @@
+package org.apache.drill.exec.store;
```

Comment:
Please move class to `org.apache.drill.exec.util`.

Reply:
Done